script/polling-heartbeat-test.js

```diff
@@ -0,0 +1,74 @@
+"use strict";
+
+var assert, config, Plugin;
+
+assert = require("assert");
+config = require("./manual-testing-config");
+Plugin = require("../../../");
```

Comment:
Wouldn't the current working directory be the folder of the plugin?

    Plugin = require(process.cwd());